### ISSUE TITLE
Allow users to switch between individual vs. bulk upload mode

### ISSUE THREAD

**Issue by lg-king** [CDCgov/prime-simplereport#4761]
## Description
Add a link to the profile dropdown that allows a user to switch between individual reporting and bulk result uploading. 

https://www.figma.com/file/NPSPM5SBzQ7Ra3mmr9M9cB/SimpleReport-Public-Site?node-id=13856%3A35434&t=qx3FidfB9GcS4q8B-1

## Design considerations: 
- What is the icon used to switch between the 2 modes? (consult with Kenny)
- What label do we use to refer to each mode?
- How to handle facility selection if they switch back to individual mode

## Acceptance criteria 
- User can switch between the bulk uploader mode and individual entry mode via a link in the profile dropdown
- We default the user to the last mode selected

Do not merge until all 1.0 changes are implemented.
- Use support admin permission until all changes are made

**Comment by mindyatwork** [comment]
Closing as this will be superseded by the new CSV bulk upload work.
